docs/track-metrics-using-kubeflow.md

```diff
@@ -0,0 +1,72 @@
+# Experiment tracking using Kubeflow Pipelines
+
+In this guide you will see how you can use kubeflow pipelines to track metrics from different experiments and runs.
+
+[Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/) is a platform for building and deploying scalable machine learning (ML) workflows. It allows us to automate the running of the jupyter notebooks and scripts using a simple workflow. Using Kubeflow pipelines we can automate the running of multiple steps of a ML pipeline like data collection, model training, inference, which need to be performed sequentially or parallel.
+ 
+Kubeflow pipelines also allow us to [export metrics](https://www.kubeflow.org/docs/components/pipelines/sdk/pipelines-metrics/) from a running pipeline which can help compare multiple runs for example over multiple model training runs. We can compare the achieved model performance metrics like accuracy, f1 score achieved.
+
+## Pre-requisites
+
+1. Existing kubeflow pipeline with components (notebooks or scripts) for which you want to track metrics.
+
+## Track Metrics
+
+To enable tracking of metrics during the execution of a machine learning experiment, your component must have an output called `mlpipeline-metrics`  which must return a JSON-serialized metrics dictionary. Let's demonstrate this with the help of this [notebook](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo1/demo1-create-tables.ipynb).
```

Comment:
Can we remove the link to that notebook here?  Move it to the very bottom of the doc, add something like, "if you want to see an real example of how to integrate experiement tracking, take a look at this noteboook"

Reply:
sounds good.